### ISSUE TITLE
ImportError when generating html docs

### ISSUE THREAD

**Issue by thopiekar** [kivy/kivy#3884]
Just checked the daily builds and noticed they are broken. On html generation it is trying to import PyInstaller, but there is no PyInstaller on Ubuntu. Is it Windows-specific?
```
mkdir -p build/html build/doctrees
python autobuild.py silenced=yes
[WARNING           ] [Config      ] Older configuration version detected (0 instead of 14)
[WARNING           ] [Config      ] Upgrading configuration in progress.
[INFO              ] [Logger      ] Record log in /home/buildd/.kivy/logs/kivy_15-12-30_0.txt
[INFO              ] [Kivy        ] v1.9.1-dev
[INFO              ] [Python      ] v2.7.11 (default, Dec  9 2015, 00:29:25) 
[GCC 5.3.1 20151207]
[INFO              ] [Factory     ] 179 symbols loaded
Purge log fired. Analysing...
Purge finished!
 Traceback (most recent call last):
   File "autobuild.py", line 59, in <module>
     import kivy.tools.packaging.pyinstaller_hooks
   File "/«BUILDDIR»/kivy-1.9.1-dev~daily0+201512301032-3264/kivy/tools/packaging/pyinstaller_hooks/__init__.py", line 72, in <module>
     from PyInstaller.compat import modname_tkinter
 ImportError: No module named PyInstaller.compat
Makefile:80: recipe for target 'html' failed
```
* https://launchpadlibrarian.net/232298007/buildlog_ubuntu-xenial-amd64.kivy_1.9.1-dev~daily0%2B201512301032-3264-testing100~ubuntu16.04.1_BUILDING.txt.gz

**Comment by dessant** [comment]
@thopiekar it is needed for the windows pyinstaller docs, you'd have to install pyinstaller master for it to work, this was a recent change.
https://github.com/kivy/kivy/blob/c55ba1171c70992b0e8eae17450c81a96bc9d5f0/.travis.yml#L48

**Comment by akshayaurora** [comment]
We should actually skip the import for pyinstaller for docs.

**Comment by dessant** [comment]
+1

**Comment by thopiekar** [comment]
At least when PyInstaller isn't installed.

**Comment by dessant** [comment]
@matham ping

**Comment by akshayaurora** [comment]
closed by 01317a1
